scripts/joinmarket-qt.py

```diff
@@ -2411,8 +2422,21 @@ def onTabChange(i):
 tabWidget.currentChanged.connect(onTabChange)
 
 mainWindow.show()
+print("the reactor object we're about to run is: {}".format(reactor))
```

Comment:
nit - this isn't some debug leftover?

Reply:
thanks, fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1132/commits/cf3763904927a99fdcaa04d8717626b1da6cce13